docs/proposals/odo-v3-cli/odo-describe-component.md

```diff
@@ -0,0 +1,35 @@
+
+# `odo describe component`
+
+Show detailed information about component.
+
+By default, it shows detail about component in the current directory.
```

Comment:
Define "detail". Define it clearly. Please.

Reply:
work in progress. Will do it later

Reply:
OK. Please mark as resolved when it's done. Thanks.

Reply:
It would be nice if you could help me with that.

Reply:
Sure.